scripts/build.sh

```diff
@@ -22,5 +22,8 @@ done
 test -z "$(go fmt ./...)" || (echo "Run go fmt ./... and recommit your code";exit 1)
 go get -u github.com/golang/protobuf/protoc-gen-go
 go generate ./...
-go install ./...
-go test $RACE_ENABLED ./... -cover
+go install cmd/...
+go install controllers/...
```

Comment:
Do we not want to do `go install plugins/...` here as well?

Reply:
plugins would be pulled in as they are imported, I thought explicitly installing them is not needed. but I can surely add them.